ShoppingM8-be/src/main/java/edu/pw/shoppingm8/list/ListServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package edu.pw.shoppingm8.list;
+
+import org.springframework.stereotype.Service;
+
+import edu.pw.shoppingm8.authentication.AuthenticationService;
+import edu.pw.shoppingm8.list.api.dto.ListDto;
+import edu.pw.shoppingm8.list.exception.ListNotFoundException;
+import edu.pw.shoppingm8.user.User;
+import edu.pw.shoppingm8.user.UserService;
+
+@Service
+public class ListServiceImpl implements ListService {
+    private final ListRepository listRepository;
+    private final UserService userService;
+    private final AuthenticationService authenticationService;
+
+    public ListServiceImpl(ListRepository listRepository,
```

Comment:
No need to write this constructor on your own - you can use lombok @RequiregArgsConstructor. It will generate a constructor that covers all final fields

Reply:
I will change it.